package.json

```diff
@@ -79,6 +80,7 @@
     "esprima": "^4.0.0",
     "expect": "^26.3.0",
     "formidable": "^1.2.1",
+    "micromatch": "^4.0.2",
```

Comment:
We try to not pull non-dev dependencies into the system. Check out `globToRegex` in Playwright, should we reuse/copy it instead?

Reply:
Minimatch/micromatch implement bash glob patterns. Minimatch (20M weekly downloads) is used by npm itself, mocha, eslint and Micromatch (24M weekly downloads) is a smaller competitor used by Jest, webpack, and others. This is the defacto standard glob syntax for the node world. `globToRegex` is missing standard features like '!' negation. I also do not trust it to work with files, and don't think it is worth any time.

Micromatch is already included in our dependency tree via `expect`.

Reply:
Okay